fotoapparat/src/main/java/io/fotoapparat/parameter/camera/CameraParameters.kt

```diff
@@ -0,0 +1,33 @@
+package io.fotoapparat.parameter.camera
+
+import io.fotoapparat.hardware.CameraDevice
+import io.fotoapparat.parameter.*
+import io.fotoapparat.util.lineSeparator
+import io.fotoapparat.util.wrap
+
+
+/**
+ * Parameters of [CameraDevice].
+ */
+data class CameraParameters(
+        val flashMode: Flash,
+        val focusMode: FocusMode,
+        val jpegQuality: Int,
+        val previewFpsRange: FpsRange,
+        val antiBandingMode: AntiBandingMode,
+        val sensorSensitivity: Int?,
+        val pictureResolution: Resolution,
+        val previewResolution: Resolution
+) {
+    override fun toString(): String {
+        return "CameraParameters" + lineSeparator +
+                "flashMode:" + flashMode.wrap() +
+                "focusMode:" + focusMode.wrap() +
+                "jpegQuality:" + jpegQuality.wrap() +
+                "previewFpsRange:" + previewFpsRange.wrap() +
+                "antiBandingMode:" + antiBandingMode.wrap() +
+                "sensorSensitivity:" + sensorSensitivity?.wrap() +
```

Comment:
FYI, a new line isn't added if the value is null so the display doesn't look quite right:

> Camera CameraParameters
> flashMode:		io.fotoapparat.parameter.Flash$Torch@743aeb
> focusMode:		io.fotoapparat.parameter.FocusMode$ContinuousFocusPicture@626481d
> jpegQuality:		90
> previewFpsRange:		FpsRange(min=15000, max=30000)
> antiBandingMode:		io.fotoapparat.parameter.AntiBandingMode$Auto@3644919
> **sensorSensitivity:nullpictureResolution:		Resolution(width=1280, height=720)**
> previewResolution:		Resolution(width=1280, height=720)

Reply:
Thanks, will fix it.